### PR TITLE
sort results on results page

### PR DESCRIPTION
**Pull request by MohamedOmarii**
related to #125

### CODE REVIEW

File: database/db_functions.js [hunk 1]
```diff
@@ -51,15 +51,33 @@ const showDb = (Model) => {
...
-      {'category': {$in: option.category}}
-    ]
-  }, (err, result) => {
+  Model.aggregate([
+    {$match: {$and: [
+    {'accessOptions': {$in: option.accessOptions}},
+    {'category': {$in: option.category}}
+    ]}},
+  {$unwind: '$accessOptions'},
+  {$unwind: '$category'},
```
Comment (m4v15) [inline database/db_functions.js]: indentation

File: routes/results.js [hunk 2]
```diff
@@ -1,17 +1,29 @@
...
+  }
+  // format the service choices
+  const serviceOption = req.query.services
+  let serviceQueries
+  if (serviceOption) {
+    serviceQueries = convertFunctions.capitalise(serviceOption).split(',')
+  } else {
+    serviceQueries = ['Food', 'Sport', 'Education', 'Health', 'Municipal', 'Fashion', 'Construction', 'IT', 'Tourism']
+  }
 // fake queries to use this data from mockData
```
Comment (m4v15) [inline routes/results.js]: delete this comment plz

Comment (MohamedOmarii) [inline routes/results.js]: :+1:

File: database/db_functions.js (hunk 1 above)
Comment (MohamedOmarii) [inline database/db_functions.js]: :+1:

**Pull request merged**